node/templates/bird.cfg.template

```diff
@@ -46,24 +49,25 @@ template bgp bgp_template {
                     # local address as nexthop
   source address {{getenv "IP"}};  # The local address we use for the TCP connection
   add paths on;
+  graceful restart;
```

Comment:
Worth mentioning the route flap bug here, too -- in case someone is tempted to remove the GR flag in the future.